torch/onnx/utils.py

```diff
@@ -1598,7 +1597,7 @@ def _export(
                     model_file_location,
                     node_attr_to_name,
                 )
-            proto = _add_onnxscript_fn(proto)
+            proto = _add_onnxscript_fn(proto, graph, custom_opsets)
```

Comment:
If a user exports two different models that use different functions, would this add all functions to the second model?

Reply:
I don't think so. There is a logic inside _add_onnxscript_fn making sure only the model-related function is inserted.

Reply:
Although a complicated test case may be added to make sure this won't happen.